src/common/config.go

```diff
@@ -114,8 +111,13 @@ service:
           - .metadata.annotations."opslevel.com/ignore"
 `
 
-func GetConfig(data string) (*Config, error) {
+func ParseConfig(data string) (*Config, error) {
 	var output Config
-	err := yaml.Unmarshal([]byte(data), &output)
-	return &output, err
+	if err := yaml.Unmarshal([]byte(data), &output); err != nil {
+		return nil, err
+	}
+	if err := defaults.Set(&output); err != nil {
```

Comment:
`default` tags are not used here at all. But the call existed to `defauls.Set()` before and I'm keeping it because we might use defaults in the future.